src/control/cmd/daos_agent/start.go

```diff
@@ -75,23 +82,34 @@ func (cmd *startCmd) Execute(_ []string) error {
 		cmd.Debug("Local fabric interface caching has been disabled")
 	}
 
+	cmd.Debug("initializing hardware providers")
```

Comment:
Logging both start AND end with timing info is a hair on the noisy side, IMO. This isn't to show progress to the user, who doesn't necessarily have access to the code in order to know what should come next after the last debug log entry. I know that I make a big deal about debug logging, but it does add up over time and it's less painful to carefully consider now what's going to be relevant in a year or whatever when the logs are full of noise and it's more difficult than it should be to pick out the useful signal.

I was also wondering if maybe we should try to DRY this stuff up with some helpers in e.g. `common/time_utils.go`, but I'm not sure there's a good way to do it without some convoluted scheme using code blocks. @tanabarr added some nice timing helpers over in the [server code](https://github.com/daos-stack/daos/blame/master/src/control/server/server.go#L162), but those are for timing a whole method.

Reply:
I had thought about doing something more clever with helpers, but came to the same conclusion. It felt convoluted. I can remove the start messages.